### PR TITLE
fix(robot-server): Fixup failing tests in robot-server

### PR DESCRIPTION
**Pull request by Laura-Danielle**
## overview

Robot-server tests were not running in CI which caused us to fail to notice that tests were not passing.

## changelog

- Fixup failling tests
- Add robot server tests to run when `make test-py` is called

## review requests

Ensure tests still look ok.

## risk assessment
None, just tests

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -121,6 +121,7 @@ test-py-windows:
 .PHONY: test-py
 test-py: test-py-windows
 	$(MAKE) -C update-server test
+	$(MAKE) -C robot-server test
```
Comment (amitlissack) [inline Makefile:124]: What fool took this out ? :(

**Review by amitlissack — APPROVED** [review]
Nice job!

Ran `make test-py` from monorepo and all tests ran and passed.

**Review by sfoster1 — APPROVED** [review]
whoops!

File: robot-server/tests/service/routers/test_session.py [hunk 2]
```diff
@@ -69,11 +69,9 @@ def mock_cal_session(hardware, loop):
             'channels': 1}
     }
 
-    def fake_get_pip(mount):
-        return other_instrs[mount]
-
-    CheckCalibrationSession._key_by_uuid = MagicMock(return_value=mock_instrs)
-    CheckCalibrationSession.get_pipette = MagicMock(side_effect=fake_get_pip)
+    CheckCalibrationSession._get_pip_info_by_mount =\
+        MagicMock(return_value=mock_instrs)
```
Comment (b-cooper) [inline robot-server/tests/service/routers/test_session.py:73]: nit: `mock_instrs` is really mocked PipetteInfo instances keyed by mount so maybe `mock_pipette_info_by_mount` would be clearer.  Similarly, `other_instrs` is actually mocked instances of the AttachedPipette class from the hardware controller so maybe `mock_hw_pipettes`?

**Review by b-cooper — APPROVED** [review]
Looks good, just noticed some naming that seems weird now. 🍨

**Pull request merged**